README.md

```diff
@@ -15,8 +15,8 @@ The Knative Serving project provides middleware primitives that enable:
 - Point-in-time snapshots of deployed code and configurations
 
 For documentation on using Knative Serving, see the
-[serving](https://github.com/knative/docs/tree/master/docs/serving) folder of
-the [Knative Docs](https://github.com/knative/docs) repository.
+[serving](https://www.knative.dev/docs/serving/) folder of the
```

Comment:
The wording around this needs to be changed.

Reply:
Wording updated. Thanks, @dgerd !